test/unit/test_autosubmit_helper.py

```diff
@@ -18,9 +18,9 @@
 """Test file for ``autosubmit_helper.py``."""
 
 import datetime
-import pytest
 from datetime import timedelta
 from typing import Callable
+import pytest
```

Comment:
`pytest` is not stdlib, so I think it would be placed in one more line below (i.e. separated from stdlib imports and from autosubmit imports). I checked the other places where you changed imports, and my IDE produced no change when I optimized imports :clap: . Except for this one.